src/main/java/org/truffleruby/core/CoreLibrary.java

```diff
@@ -525,7 +529,11 @@ public CoreLibrary(RubyContext context) {
         defineModule(truffleModule, "ReadlineHistory");
         defineModule(truffleModule, "ThreadOperations");
         psychModule = defineModule("Psych");
-        psychParserClass = defineClass(psychModule, objectClass, "Parser");
+        publicPsychParserClass = defineClass(psychModule, objectClass, "Parser");
+        psychYAMLParserClass = defineClass(psychModule, objectClass, "YAMLParser");
```

Comment:
I think those should be defined under `Psych::Parser`, not under `Psych` directly.

Reply:
I'll move them.